modules/post/windows/gather/enum_av.rb

```diff
@@ -0,0 +1,50 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::Priv
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Installed AntiVirus Enumeration',
+        'Description' => %q{
+          This module will enumerate the AV products detected by WMIC
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'rageltman <rageltman[at]sempervictus>' ],
+        'Platform' => %w[win],
+        'SessionTypes' => [ 'meterpreter', 'shell' ],
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [],
+          'SideEffects' => []
+        }
+      )
+    )
+  end
+
+  # Run Method for when run command is issued
+  def run
+    if command_exists?('wmic') == false
+      print_error("The 'wmic' command doesn't exist on this host!") # wmic is technically marked as depreciated so this command could very well be removed in future releases.
+      return
+    end
+    avs = {}
+    cmd = 'wmic /namespace:\\\\root\\SecurityCenter2 path AntiVirusProduct get * /value'
+    resp = cmd_exec(cmd, nil, 6000).to_s
+    fail_with(Failure::Unknown, resp) if resp[0..5] == 'ERROR:'
```

Comment:
Will this work consistently when the language pack is no english?

Reply:
Perhaps but looks like it might have to be case insensitive

```
C:\Users>wmic /namespace:\\\\root\\SecurityCenter2 path AntiVirusProduct get * /value
Error:
Descripción = El servidor RPC no está disponible.

C:\Users>
```

Reply:
![image](https://user-images.githubusercontent.com/63261883/163650194-1bb83767-d6bd-431a-9935-176af56dfaa7.png)

Seems to all be the same for most systems minus that Error part potentially being in lower case.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/16432/commits/ce76f0e62e1e182930a4cf0626111512298fcbb5